src/pages/maps/index.js

```diff
@@ -96,7 +103,7 @@ function Maps() {
                             }
                             return (
                                 <div className="map-wrapper" key={`map-wrapper-${key}`}>
-                                    <h3>{displayText}</h3>
+                                    <h3>{map.projection}</h3>
```

Comment:
```suggestion
                                    <h3>{displayText}</h3>
```
There should be `displayText` here to show different map variants.

Reply:
Reverted, is there a way to get the projection i.e - 2D / 3D without the map name? That was my aim :)

Reply:
`displayText` is a concatenation of 5 fields: `name`, `projections`, `orientation`, `specific`, `extra`. We use it in quite many places to specify what map it is. Eg: for the title of the page, alt name of the thumb, map name in the remote control and some other minor places.
But we can think of removing the `name` and just adding it manually where needed.